webrender_traits/src/api.rs

```diff
@@ -67,6 +69,15 @@ impl RenderApi {
         key
     }
 
+    ///
+    pub fn get_glyph_dimensions(&self, font_key: FontKey, size: Au, blur_radius: Au, index: u32)
```

Comment:
I wonder if this should be able to pass multiple glyphs to get in one batch, to reduce the overhead of an IPC message per glyph?